scripts/test_sandbox.sh

```diff
@@ -66,8 +68,15 @@ sandbox_pid=$!
 echo "💤 Waiting for sandbox . . . (${sandbox_pid})"
 
 function finish {
-  kill -9 $sandbox_pid
-  echo "💀 Killed process ${sandbox_pid}"
+    if [ $interactive -eq 1 ]; then
+        echo "🤔 Do you want to stop the sandbox (${sandbox_pid})? (Y/n)"
+        read -r proceed
+        if [ "$proceed" == "n" ]; then
+            exit 0
```

Comment:
should we include the same warning related to PID in sandbox mode?

Reply:
Done